templates/wizards/projects/asn1acn/handleAsn1AcnBuild.pri

```diff
@@ -22,7 +22,7 @@
 #######################################################################
 
 isEmpty(ASN1SCC): ASN1SCC = $$(ASN1SCC)
-isEmpty(ASN1SCC): ASN1SCC = asn1.exe
+isEmpty(ASN1SCC): ASN1SCC = asn1scc
```

Comment:
again, why?

I mean, we can, but we would need to repack the asn1scc distributable first

Reply:
explanation why is above
reverted